### ISSUE TITLE
State of the project?

### ISSUE THREAD

**Issue by maxwondercorn** [Glavin001/ember-c3#74]
I love this project but see that it hasn't been maintained for a while.  @Glavin001 if you don't have the opportunity to maintain the project, I'd like to create a new project based on this one.  Minimally, I would move it to the latest ember LTS and move from Bower to NPM for the C3 dependency.

**Comment by maxwondercorn** [comment]
Yes, I figured you were busy elsewhere as happens to us all.  Yes, it would be great to have access here. 👍  I would rather extend what you've created, plus I could start looking the existing PRs

**Comment by Glavin001** [comment]
I've added your GitHub username `maxwondercorn` to the list of GitHub collaborators. Please let me know if you need access to anything else. Thank you for your interest and support! 😃
